package.json

```diff
@@ -8,31 +8,38 @@
     "email": "stevehansen@geotab.com"
   },
   "devDependencies": {
-    "autoprefixer": "^6.2.3",
-    "babel-preset-es2015": "^6.9.0",
-    "browser-sync": "^2.11.0",
-    "del": "^2.2.0",
+    "autoprefixer": "^9.8.0",
+    "babel-preset-es2015": "^6.24.1",
```

Comment:
The `es2015` preset was giving me an error running locally. Looks like its incompatible with `@babel/core` v7. [Replacing with @babel/preset-env](https://github.com/babel/gulp-babel/issues/165#issuecomment-443102581) solved the issue for me. Had to update in `gulp/tasks/assets.js` as well.